bridge/opencensus/exporter.go

```diff
@@ -115,36 +115,36 @@ func (d *metricReader) ForEach(_ aggregation.TemporalitySelector, f func(export.
 	return nil
 }
 
-// convertLabels converts from OpenCensus label keys and values to an
-// OpenTelemetry label Set.
-func convertLabels(keys []metricdata.LabelKey, values []metricdata.LabelValue) (attribute.Set, error) {
+// convertAttrs converts from OpenCensus attribute keys and values to an
+// OpenTelemetry attribute Set.
+func convertAttrs(keys []metricdata.LabelKey, values []metricdata.LabelValue) (attribute.Set, error) {
 	if len(keys) != len(values) {
-		return attribute.NewSet(), fmt.Errorf("%w different number of label keys (%d) and values (%d)", errConversion, len(keys), len(values))
+		return attribute.NewSet(), fmt.Errorf("%w different number of attribute keys (%d) and values (%d)", errConversion, len(keys), len(values))
```

Comment:
Should this remain `label` as it is referring to the OpenCensus `LabelKey` and `LabelValue` types?